src/models/train_model.py

```diff
@@ -0,0 +1,36 @@
+"""
+Train models for predictions.
+"""
+
+from joblib import dump, load
+from define_model import _get_parameters
```

Comment:
For this I would add a __init__.py file to the subfolders and do models.define_model